tools/oeutil/README.md

```diff
@@ -40,35 +40,40 @@ where `options` are:
 
 Note that parameters are not case-sensitive.
 
-Example 1 Generate, verify and dump a ceritificate. Without "--out", there will be no certificate output file:
+Example 1. Generate, verify and dump a ceritificate. Without "--out", there will be no certificate output file:
 
     ./oeutil generate-evidence --format cert keyecec.pem publickeyec.pem --verify --verbose
 
-Example 2 Generate an OE report and output its endorsements binary to "endorsements.bin", and output OE report to "report.bin":
+Example 2. Generate an OE report and output its endorsements binary to "endorsements.bin", and output OE report to "report.bin":
 
     ./oeutil generate --format legacy_report_remote --endorsements endorsements.bin --out report.bin
 
-Example 3 Generate and verify OE evidence in SGX_ECDSA format, dump evidence buffer and its verified claims, and output OE evidence to "evidence.bin":
+Example 3. Generate and verify OE evidence in SGX_ECDSA format, dump evidence buffer and its verified claims, and output OE evidence to "evidence.bin":
 
     ./oeutil gen --format sgx_ecdsa --verify --verbose --out evidence.bin
 
+-----
 
-Creating RSA and EC keys pairs in Linux using openssl.
-Generate an RSA private key, of size 2048, and output it to a file named key.pem
-=====
-openssl genrsa -out keyrsa.pem 2048
+## Using OpenSSL to create a key pair on Linux
```

Comment:
I think "on Linux" should be removed.  openssl is cross-platform.

Reply:
Thanks! Just verified all the commands again and removed it.